clients/drcachesim/tests/view_test.cpp

```diff
@@ -586,11 +586,74 @@ run_chunk_tests(void *drcontext)
     return run_single_thread_chunk_test(drcontext) && run_serial_chunk_test(drcontext);
 }
 
+/* Test view tool on a OFFLINE_FILE_TYPE_ARCH_REGDEPS trace.
+ * The trace hardcoded in entries is X64, so we only test on X64 architectures here.
+ */
+bool
+run_regdeps_test(void *drcontext)
+{
+#ifdef X64
+    const memref_tid_t t1 = 3;
+    std::vector<memref_tid_t> tids = { t1 };
+    constexpr addr_t PC = 0x00007f6fdd3ec360;
+    constexpr addr_t PC_vpmovmskb = 0x00007f6fdc76cb35;
+    constexpr addr_t PC_vmovdqu = 0x00007f6fdc76cb2d;
+    constexpr addr_t ENCODING_REGDEPS_ISA = 0x0006090600010011;
```

Comment:
This should have a suffix to indicate it's a particular instruction, like the ones below.  Otherwise it looks like it's referring to some generic ISA value.

Reply:
Done. Also added a two line encoding instruction to the test.